cypress/e2e/real-time-events/advanced-filter.cy.js

```diff
@@ -1,21 +1,21 @@
 /* eslint-disable cypress/no-unnecessary-waiting */
 import selectors from '../../support/selectors'
 
-describe('Real-Time Events Advanced Filter', { tags: ['@dev3'] }, () => {
+describe('Real-Time Events Advanced Filter', { tags: ['@dev3', '@skip'] }, () => {
```

Comment:
Don't you mean `@xfail`?
https://github.com/aziontech/azion-console-kit/blob/c920e058cbbd95f22978292a0663bec0013a6882/.github/workflows/pre-merge.yml#L73

Pretty sure adding this change here will be as useful as a sand in the desert 😅